cmake/utils.cmake

```diff
@@ -203,7 +222,7 @@ function(mayaUsd_copyFiles target)
         message(FATAL_ERROR "DESTINATION keyword is not specified.")
     endif()
```

Comment:
mayaUsd_promoteHeaderList function now takes a TARGET, DIRNAME, and HEADERS which are needed to be passed explicitly.